first_hour_on_terra/working_with_data_collections.ipynb

```diff
@@ -227,34 +234,42 @@
    "source": [
     "### Convert workspace to data collection\n",
     "\n",
-    "Run the cell below to change the type of the workspace you created to 'data collection'. The output should resemble:\n",
+    "Now you'll convert your newly created workspace, to which you have added resources, into a data collection which can be shared with others and added to other workspaces. \n",
+    "Run the cell below to create a widget, then populate the widget's input fields and click the button to convert the workspace to a data collection.\n",
+    "\n",
+    "Widget input parameters include:\n",
+    "- `workspace_id`: Should match the workspace ID of the workspace created above.\n",
```

Comment:
The workspace_id HAS to match, right? "Must match" instead of Should match.

Reply:
Fixed!